### PR TITLE
Add whitespace section

### PR DESCRIPTION
**Pull request by felixklauke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,32 @@ void defeatDragon(Dragon dragon) {
...
+    return name;
+  }
+  
+}
+
+// GOOD
+public class Person {
+  private String name;
+
+  publis String getName() {
```
Comment (ehenoma) [inline README.md]: You spelled `public` wrong.

File: README.md [hunk 2]
```diff
@@ -53,6 +53,32 @@ void defeatDragon(Dragon dragon) {
...
+  public String getName() {
+    
+    return name;
+  }
+  
+}
+
+// GOOD
+public class Person {
+  private String name;
```
Comment (ehenoma) [inline README.md]: Name should be `final`

File: README.md [hunk 3]
```diff
@@ -53,6 +53,32 @@ void defeatDragon(Dragon dragon) {
 #### Max line length
 Prefer to keep the amount of characters below `80` and hard wrap lines at `90` characters.
 
+#### White spaces
+Avoid using whitespaces at the beginning and at the end of class and method definitions to reduce the vertical size of our code.
```
Comment (ehenoma) [inline README.md:57]: That's good!

**Review by ehenoma — CHANGES_REQUESTED** [review]
Please fix the `public` keyword, then I'll merge.

File: README.md (hunk 1 above)
Comment (felixklauke) [reply README.md]: Thats already fixed. Look at the latest "Fix typo" commit.

File: README.md (hunk 2 above)
Comment (felixklauke) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (felixklauke) [reply README.md:57]: Yep.

**Review by ehenoma — APPROVED** [review]

**Pull request merged**